src/containers/pages/JurisdictionAssignment/helpers/Slider/index.tsx

```diff
@@ -0,0 +1,175 @@
+/** renders the range slider component that is used to set the
+ * threshold from which we can filter out the jurisdictions to be
+ * auto-targeted based on their risk value
+ */
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import reducerRegistry from '@onaio/redux-reducer-registry';
+import React, { useState } from 'react';
+import InputRange, { Range } from 'react-input-range';
+import 'react-input-range/lib/css/index.css';
+import { connect } from 'react-redux';
+import { Button } from 'reactstrap';
+import Col from 'reactstrap/lib/Col';
+import Row from 'reactstrap/lib/Row';
+import { ActionCreator, Store } from 'redux';
+import {
+  ADJUST_SLIDER_MESSAGE,
+  CONTINUE_TO_NEXT_STEP,
+  NUMBER_OF_STRUCTURES_IN_JURISDICTIONS,
+} from '../../../../../configs/lang';
+import { PlanDefinition } from '../../../../../configs/settings';
+import { RISK_LABEL } from '../../../../../constants';
+import hierarchyReducer, {
+  autoSelectNodes,
+  AutoSelectNodesAction,
+  FetchedTreeAction,
+  fetchTree,
+  getStructuresCount,
+  getTreeById,
+  reducerName as hierarchyReducerName,
+} from '../../../../../store/ducks/opensrp/hierarchies';
+import { TreeNode } from '../../../../../store/ducks/opensrp/hierarchies/types';
+import { selectionReason } from '../../../../../store/ducks/opensrp/hierarchies/utils';
+import jurisdictionMetadataReducer, {
+  fetchJurisdictionsMetadata,
+  FetchJurisdictionsMetadataAction,
+  getJurisdictionsMetadata,
+  reducerName,
+} from '../../../../../store/ducks/opensrp/jurisdictionsMetadata';
+import './slider.css';
+
+reducerRegistry.register(reducerName, jurisdictionMetadataReducer);
+reducerRegistry.register(hierarchyReducerName, hierarchyReducer);
+
+/** props for JurisdictionSelectionSlider */
+interface Props {
+  rootJurisdictionId: string;
+  tree?: TreeNode;
+  structuresCount: number;
+  fetchJurisdictionsMetadataCreator: ActionCreator<FetchJurisdictionsMetadataAction>;
+  jurisdictionsMetadata: any[];
+  autoSelectCreator: ActionCreator<AutoSelectNodesAction>;
+  fetchTreeCreator: ActionCreator<FetchedTreeAction>;
+  plan: PlanDefinition | null;
+  onClickNext: any;
```

Comment:
if this is supposed to be a function could this be refactored to `onClickNext: () => void`